google/cloud/storage/blob.py

```diff
@@ -1024,9 +1023,7 @@ def _do_download(
         :type retry: google.api_core.retry.Retry
         :param retry: (Optional) How to retry the RPC. A None value will disable
             retries. A google.api_core.retry.Retry value will enable retries,
-            and the object will configure backoff and timeout options. Custom
-            predicates (customizable error codes) are not supported for media
-            operations such as this one.
+            and the object will configure backoff and timeout options.
```

Comment:
nit: Do we want to call out the default retry? I recall we received some feedback around stating the default value in our docstrings

Reply:
I would like to incorporate DEFAULT_RETRY into the docs somehow without copy/pasting it into every method. Not sure how to do that yet. I'd prefer not to include it in this change as it's already quite large but it's a good idea.

Reply:
Sounds good, let's update that in a separate PR